templates.dist/estate/default_detail.php

```diff
@@ -18,103 +17,112 @@
-				</li>
-			<?php endif; ?>
-		</ul>
-
-	<?php endforeach; ?>
-
+<div class="oo-detailview">
 	<?php
+	$pEstates->resetEstateIterator();
+	while ( $currentEstate = $pEstates->estateIterator() ) { ?>
+		<div class="oo-detailsheadline">
+			<h1><?php echo $currentEstate["objekttitel"]; ?></h1>
+		</div>
+		<div class="oo-details-main">
+			<div class="oo-detailsgallery" id="oo-galleryslide">
+				<?php $estatePictures = $pEstates->getEstatePictures();
+				foreach ( $estatePictures as $id ) { ?>
+				<div class="oo-detailspicture" style="background-image: url('<?php echo $pEstates->getEstatePictureUrl( $id ); ?>');"></div>
+				<?php } ?>
+			</div>
+			<div class="oo-detailstable">	
+				<?php foreach ( $currentEstate as $field => $value ) {
+					if ( is_numeric( $value ) && 0 == $value ) {
+						continue;
+					}
+					if ( $field === "objekttitel" || $field === "objektbeschreibung" || $field === "lage" || $field === "ausstatt_beschr" || $field === "sonstige_angaben" ) {
+						continue;
+					}
+					if ( $value == "" ) {
+						continue;
+					}
+					echo '<div class="oo-detailslisttd">'.esc_html($pEstates->getFieldLabel( $field )) .'</div><div class="oo-detailslisttd">'.(is_array($value) ? esc_html(implode(', ', $value)) : esc_html($value)).'</div>';
+				} ?>
+			</div>
+			<?php if ( $currentEstate["objektbeschreibung"] !== "" ) { ?>
+				<div class="oo-detailsfreetext">	
+					<h2><?php esc_html_e('Description', 'onoffice') ?></h2>
+					<?php echo $currentEstate["objektbeschreibung"]; ?>
+				</div>
+			<?php } ?>
+			<?php if ( $currentEstate["lage"] !== "" ) { ?>
+				<div class="oo-detailsfreetext">
+					<h2><?php esc_html_e('Location', 'onoffice') ?></h2>
+					<?php echo $currentEstate["lage"]; ?>
+				</div>
+			<?php } ?>
 
-	$estateMovieLinks = $pEstates->getEstateMovieLinks();
-	foreach ($estateMovieLinks as $movieLink) {
-		echo '<a href="'.esc_attr($movieLink['url']).'" title="'.esc_attr($movieLink['title']).'">'
-			.esc_html($movieLink['title']).'</a><br>';
-	}
-
-	$movieOptions = array('width' => 500); // optional
-
-	foreach ($pEstates->getMovieEmbedPlayers($movieOptions) as $movieInfos) {
-		echo '<h3>'.esc_html($movieInfos['title']).'</h3>';
-		echo $movieInfos['player'];
-	}
-
-	$estatePictures = $pEstates->getEstatePictures();
-	foreach ( $estatePictures as $id ) : ?>
-	<a href="<?php echo $pEstates->getEstatePictureUrl( $id ); ?>">
-		<img src="<?php echo esc_url($pEstates->getEstatePictureUrl( $id, array('width' => 300, 'height' => 400) )); ?>">
-		<?php echo $pEstates->getEstatePictureTitle($id).'<br>'; ?>
-	</a>
-	<?php endforeach; ?>
-
-	<?php if ($pEstates->getDocument() != ''): ?>
-		<h2><?php esc_html_e('Documents', 'onoffice'); ?></h2>
-		<a href="<?php echo $pEstates->getDocument(); ?>">
-			<?php esc_html_e('PDF expose', 'onoffice'); ?>
-		</a>
-	<?php endif; ?>
-	<?php echo $pEstates->getSimilarEstates(); ?>
-<?php endwhile; ?>
+			<div class="oo-detailsmap">
+				<?php // require('map/map.php'); ?>
```

Comment:
In most cases, the client has disabled the output of the address which makes the map show broken. Because of that, we do not implement it in the default.